student.rb

```diff
@@ -3,8 +3,8 @@
 class Student < Person
```

Comment:
- [ ] On line one of this file, please change the file name to `person` instead of `Person`. When I try to run your `main.rb`, I get the error as seen in the screenshot because of this. 
![image](https://user-images.githubusercontent.com/45460192/232106831-6310c730-0786-4955-81a5-4060018373d3.png)
- [ ] Please do the same thing in the `teacher.rb` file.